dcat/rdf/dcat.ttl

```diff
@@ -1594,30 +1597,31 @@ prov:wasGeneratedBy
   skos:scopeNote "Use prov:qualifiedGeneration to attach additional details about the relationship between the dataset and the activity, e.g. the exact time that the dataset was produced during the lifetime of a project."@en ;
   skos:scopeNote "Utiliza prov:qualifiedGeneration para adjuntar detalles adicionales sobre la relación entre el conjunto de datos y la actividad activity, por ejemplo, el tiempo exacto en el que se produjo el conjunto de datose durante la vida de un proyecto."@es ;
   skos:scopeNote "Pro připojení podrobností o vztahu mezi datovou sadou a aktivitou, např. přesný čas kdy byla datová sada vyprodukována během životního cyklu projektu, použijte prov:qualifiedGeneration."@cs ;
-  skos:changeNote "Property added in this context in this revision of DCAT"@en ;
-  skos:changeNote "Esta propiedad se agregó en este contexto en esta revisión de DCAT"@es ;
-  skos:changeNote "Vlastnost v tomto kontextu byla přidána v této revizi DCAT"@cs ;
+  skos:changeNote "Property added in this context in DCAT 2.0."@en ;
+  skos:changeNote "Esta propiedad agregado en DCAT 2.0."@es ;
+  skos:changeNote "Vlastnost v tomto kontextu byla přidána v revizi DCAT 2.0"@cs ;
+
 .
 foaf:homepage
   skos:scopeNote "foaf:homepage is an inverse functional property (IFP), which means that it MUST be unique and precisely identify the web-page for the resource. This property indicates the canonical web-page, which might be helpful in cases where there is more than one web-page about the resource."@en ;
   skos:scopeNote "foaf:homepage es la propiedad funcional inversa (IFP por sus siglas en inglés), lo que significa que DEBE ser única y identificar la página web del recurso percisamente. Esta propiedad indica la página web canónica, que pude ser útil en casos en los que hay más de una página web para el recurso."@es ;
   skos:scopeNote "foaf:homepage je inverzní funkčí vlastnost, což znamená, že MUSÍ být unikátní a MUSÍ přesně označovat webovou stránku zdroje. Označuje hlavní webovou stránku, což se může hodit v případě, že webových stránek o zdroji existuje více."@cs ;
 .
 time:hasBeginning
-  skos:scopeNote "Use of the property time:hasBeginning entails that value of the dct:temporal property is a member of the time:TemporalEntity class from [OWL-TIME]. In this context this could be taken to imply that dct:PeriodOfTime is equivalent to the subclass time:ProperInterval"@en ;
+  skos:scopeNote "Use of the property time:hasBeginning entails that value of the dct:temporal property is a member of the time:TemporalEntity class from [OWL-TIME]. In this context this could be taken to imply that dct:PeriodOfTime is equivalent to the subclass time:ProperInterval" ;
```

Comment:
```suggestion
  skos:scopeNote "Use of the property time:hasBeginning entails that value of the dct:temporal property is a member of the time:TemporalEntity class from [OWL-TIME]. In this context this could be taken to imply that dct:PeriodOfTime is equivalent to the subclass time:ProperInterval"@en ;
```